### PR TITLE
Add install-lsp subcommand

### PR DESCRIPTION
**Pull request by h-michael**
related https://github.com/rust-analyzer/rust-analyzer/pull/617#issuecomment-456914650

### CODE REVIEW

File: .cargo/config [hunk 1]
```diff
@@ -1,8 +1,10 @@
 [alias]
 # Automatically generates the ast and syntax kinds files
 gen-syntax   = "run --package tools --bin tools -- gen-syntax"
-# Extracts the tests from 
+# Extracts the tests from
 gen-tests    = "run --package tools --bin tools -- gen-tests"
+# Installs ra-lsp-server
+install-lsp = "run --package tools --bin tools -- install-lsp"
```
Comment (matklad) [inline .cargo/config]: I think this could be just `install --package ra_lsp_server --force`, without code changes?

**Comment by matklad** [comment]
bors delegate+

**Comment by h-michael** [comment]
@matklad sorry, re-fixed. `--package` is invalid option with install subcommand.

**Pull request merged**